examples/README.md

```diff
@@ -4,15 +4,20 @@ Here you can find notebooks showing features and example usage of pytket.
 
 *Note: When viewing our examples, github has a bug where it occasionally can't render Jupyter notebooks properly. To bypass this, you can either clone the repo and open the examples in Jupyter yourself (this is recommended! You can play around with them), or you can enter the corresponding URL into https://nbviewer.jupyter.org and view examples there.*
 
-[cirq_routing_example](https://github.com/CQCL/pytket/blob/master/examples/cirq_routing_example.ipynb) - Cirq interface and routing.
+[backends_example](https://github.com/CQCL/pytket/blob/master/examples/backends_example.ipynb) - Running circuits on our allowed backends.
```

Comment:
How about "some of our backends" rather than "our allowed backends"?

Reply:
Makes sense. Have fixed all spelling mistakes.